librz/core/cmd_descs/cmd_type.yaml

```diff
@@ -81,6 +150,28 @@ commands:
         cname: type_noreturn_del_all
         summary: Remove all noreturn references
         args: []
+  - name: to
```

Comment:
Instead of using `load` as cname and `Load` as summary for most of these, i would rephrase a bit and use the word `open`/`Open` because i guess `to` stands for `type open`, meaning that you open a file to work on types. `to` opens a file and loads types from it, `toe` open the editor, `tos` open a sdb file and load types from it.